joplin/urls.py

```diff
@@ -57,8 +58,8 @@ def reroute(request):
     url(r'^admin/', include(wagtailadmin_urls)),
     url(r'^documents/', include(wagtaildocs_urls)),
     path('__debug__/', include(debug_toolbar.urls)),
-    url(r'^api/graphql', csrf_exempt(GraphQLView.as_view())),
-    url(r'^api/graphiql', csrf_exempt(GraphQLView.as_view(graphiql=True, pretty=True))),
+    url(r'^api/graphql', jwt_token_decorator(csrf_exempt(PrivateGraphQLView.as_view()))),
```

Comment:
Will Previews need a jwt_token?

Reply:
That will be handled on this issue: https://github.com/cityofaustin/techstack/issues/4769

I wont merge this one until that one is done.